homes-ms-storesmanagement/src/main/java/com/tenx/ms/storesmanagement/orders/rest/dto/OrderProductsDTO.java

```diff
@@ -0,0 +1,34 @@
+package com.tenx.ms.storesmanagement.orders.rest.dto;
+
+
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.validation.constraints.NotNull;
+
+
+public class OrderProductsDTO {
+
```

Comment:
i do agree with you that the DTO qualifier is in the package name , but as usually you don't use the package name in most of the code (because you imported the type in the imports section) i believe that this will increase readability in the rest of the code